package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hmrc-frontend",
-  "version": "4.9.0",
+  "version": "5.0.0",
```

Comment:
Is this a major version change now? I thought we'd decided this was just a minor version change because we're providing a default.

Reply:
Is it due to the introduction of `.hmrc-page-heading--with-margin`? I'm not sure if just adding this new class needs a major version bump? Happy to discuss as I'm not 100% sold either way.

Reply:
I have changed the versioning from 5.0.0 to 4.9.1 as this is only providing a default and adding a new style `.hmrc-page-heading--with-margin` shouldn't warrant a major version bump.